atomate/utils/utils.py

```diff
@@ -311,4 +312,37 @@ def recursive_update(d, u):
 def get_a_unique_id():
     ts = "{:.4f}".format(time())
     ts += str(randint(0,9999)).zfill(4)
-    return ts
\ No newline at end of file
+    return ts
+
+
+def get_uri(dir_name):
+    """
+    Returns the URI path for a directory. This allows files hosted on
+    different file servers to have distinct locations.
+    Args:
+        dir_name:
+            A directory name.
+    Returns:
+        Full URI path, e.g., fileserver.host.com:/full/path/of/dir_name.
+    """
+    fullpath = os.path.abspath(dir_name)
+    try:
+        hostname = socket.gethostbyaddr(socket.gethostname())[0]
+    except:
+        hostname = socket.gethostname()
+    return "{}:{}".format(hostname, fullpath)
+
+
+
+def get_database(config_file=None, settings=None, admin=False, **kwargs):
```

Comment:
why move this out of pymatgen-db and into atomate?

Reply:
Nevermind I see you answered this already.

Can you remove pymatgen-db from requirements.txt and setup.py in that case?